packages/ogm/src/utils/filter-document.ts

```diff
@@ -56,7 +66,28 @@ function filterDocument(typeDefs: Neo4jGraphQLConstructor["typeDefs"]): Document
                             ...r,
                             {
                                 ...f,
-                                directives: f.directives?.filter((x) => !excludedDirectives.includes(x.name.value)),
+                                directives: f.directives
+                                    ?.filter((x) => !excludedDirectives.includes(x.name.value))
+                                    .map((x) => {
+                                        if (x.name.value === "relationship") {
+                                            return {
+                                                ...x,
+                                                arguments: x.arguments?.map((arg) => {
+                                                    if (arg.name.value === "aggregate") {
+                                                        return {
+                                                            ...arg,
+                                                            value: {
+                                                                ...arg.value,
+                                                                value: true,
+                                                            } as BooleanValueNode, // expect a BooleanValue GraphQL validation already happened
+                                                        };
+                                                    }
+                                                    return arg;
+                                                }),
```

Comment:
Looks to me like this will only work if the `aggregate` argument has already been included in the directive, but aren't we going to be setting this to `false` by default in `4.0.0`? So this will have to add the `aggregate` argument regardless of whether it was in the original input.

Reply:
Ouch, I'm pretty sure you're right here :(